experiments/src/main/kotlin/org/evomaster/experiments/linear/Main.kt

```diff
@@ -157,7 +157,7 @@ class Main {
             val manager = a.first.getInstance(LifecycleManager::class.java)
 
             manager.start()
-            val sol = a.second.search()
+            val sol = a.second.searchOnce()
```

Comment:
why the change from `search` to `searchOnce`? wouldn't that break this code, as only a single step in the search would be made?